docs/en/sql-reference/functions/string-search-functions.md

```diff
@@ -689,47 +689,47 @@ Calculates the [hamming distance](https://en.wikipedia.org/wiki/Hamming_distance
 **Syntax**
 
 ```sql
-byteHammingDistance(string2, string2)
+byteHammingDistance(string1, string2)
```

Comment:
(minor: I suppose docs for the string distance functions rather belongs into string-functions.md instead of string-search-function.md - we are not really "searching" in strings, we are doing something "with" strings)